edb/schema/pointers.py

```diff
@@ -534,6 +535,22 @@ class Pointer(referencing.NamedReferencedInheritingObject,
         default=None,
     )
 
+    rewrites_refs = so.RefDict(
+        attr="rewrites",
+        requires_explicit_overloaded=True,
+        backref_attr="subject",
+        ref_cls=s_rewrites.Rewrite,
+    )
+
+    rewrites = so.SchemaField(
```

Comment:
Add to `schema.edgeql` or not yet?

Reply:
Wasn't done in the PR yet; isn't needed yet